Integrations/MicrosoftTeams/MicrosoftTeams.yml

```diff
@@ -225,7 +225,33 @@ script:
     description: Adds a member (user) to a private channel.
     execution: false
     name: add-user-to-channel
-  dockerimage: demisto/teams:1.0.0.1701
+  - arguments:
+    - default: false
+      description: The user to call.
+      isArray: false
+      name: username
+      required: true
+      secret: false
+    - default: false
+      description: The tenant ID for the Teams.
+      isArray: false
+      name: tenant_id
+      required: true
+      secret: false
+    - default: false
+      description: The callback URI for the call. The URI determins who will recieve
```

Comment:
how will you get that as a user?

Reply:
you are right there is no way I found to get this info. you can put a bot's callback_uri if you want the call to redirect to a different bot for some reason - I'll edit the description accordingly